README.Rmd

```diff
@@ -0,0 +1,123 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+<!-- badges: start -->
+
+[![R-CMD-check](https://github.com/rstudio/rticles/workflows/R-CMD-check/badge.svg)](https://github.com/rstudio/rticles/actions) [![Downloads from the RStudio CRAN mirror](https://cranlogs.r-pkg.org/badges/rticles)](https://cran.r-project.org/package=rticles) [![CRAN status](https://www.r-pkg.org/badges/version/rticles)](https://CRAN.R-project.org/package=rticles)
+
+<!-- badges: end -->
+
+# rticles
+
+The **rticles** package provides a suite of custom [R Markdown](https://rmarkdown.rstudio.com) LaTeX formats and templates for various formats. Most of the templates are provided and maintained by the community, and anyone can contribute a new template. See [How to contribute](#how-to-contribute) below.
+
+## Book
+
+<a href="https://bookdown.org/yihui/rmarkdown/journals.html"><img src="https://bookdown.org/yihui/rmarkdown/images/cover.png" alt="R Markdown: The Definitive Guide" height="400"/></a>
+
+## Installation
+
+You can install and use **rticles** from CRAN as follows:
+
+``` r
+install.packages("rticles")
+```
+
+If you wish to install the development version from GitHub (which often contains new article formats), you can do this:
+
+``` r
+remotes::install_github("rstudio/rticles")
+```
+
+## Using rticles
+
+```{r child="man/fragments/usage.md"}
+```
+
+## Templates
+
+Currently included templates and their contributors are the following:
+
+| Journal                                                                                                                                                                           | Contributors                                                                                                                                                                   | Pull request                                                                                                                                                                                                                                                                                                                     | Output format              |
```

Comment:
I am not sure why the table got messed up formating in the source file. There are a lot of spaces introduced which will make it very hard to modify in the future. 

Can I try fix this or was it on purpose ? Did you use visual editor maybe ? 

Having this done another way is also an option: #306

Reply:
When I copied the current README.md to README.Rmd the table was a mess when I built the pkgdown site. I tried using the visual editor to fix it, and it did, but you are welcome to futz with this!

Reply:
I'll format in the README as it was before so that it is still 'easy' to modify. 
I'll need to find a new way of managing this table I think. 

Thanks!